distro.py

```diff
@@ -991,8 +991,6 @@ def uname_info(self):
         """
         Return a dictionary containing key-value pairs for the information
         items from the uname command data source of the OS distribution.
-
-        For details, see :func:`distro.uname_info`.
```

Comment:
my question is whether the function `uname_info` shall rather appear in the docs.